### PR TITLE
Adding CircleCI job to trigger UI tests on deployment

### PR DESCRIPTION
**Pull request by adamhughes86**

### CODE REVIEW

**Comment by adamhughes86** [comment]
I've ran a local test in a different environment and this _can_ work. It's setup a bit different for this project though so the trigger may not work on this first pass. I'll monitor the build when it's merged though to see what happens

File: .circleci/config.yml [hunk 1]
```diff
@@ -72,6 +95,9 @@ workflows:
             - hmpps-approved-premises-api-prod
           requires:
             - request-prod-approval
+      - trigger-ui-tests:
+          requires:
+            - request-prod-approval
```
Comment (edavey) [inline .circleci/config.yml:100]: I'm not clear why these tests require prod approval? My understanding was that they run in the dev environment.

Comment (adamhughes86) [reply .circleci/config.yml:100]: I was thinking that the api deploys to prod and then the UI tests are run to validate the latest

Comment (adamhughes86) [reply .circleci/config.yml]: I've changed this to a deploy_dev requirement now. I've tested it by adding the feature branch to the filtered branches list and it triggers the UI builds once the api is deployed to dev

**Review by edavey — APPROVED** [review]
thanks a lot for figuring this out!

**Pull request merged**